scm-webapp/src/test/java/sonia/scm/net/ahc/DefaultAdvancedHttpResponseTest.java

```diff
@@ -130,13 +140,10 @@ public void testGetHeaders()
     assertTrue(headers.get("Test-2").isEmpty());
   }
 
-  //~--- fields ---------------------------------------------------------------
-
-  /** Field description */
-  private final DefaultAdvancedHttpClient client =
-    new DefaultAdvancedHttpClient(new ScmConfiguration(), new HashSet<>(), new SSLContextProvider());
-
   /** Field description */
   @Mock
   private HttpURLConnection connection;
+
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+  private Tracer tracer;
```

Comment:
Especially here you move the fields to top. One field is at top and the others at the bottom...